src/Whoops/Util/Misc.php

```diff
@@ -41,4 +41,21 @@ public static function translateErrorCode($error_code)
         }
         return "E_UNKNOWN";
     }
+    
+    /**
+     * Determine if an error level is fatal (halts execution)
+     * 
+     * @param int $level
+     * @return boolean
```

Comment:
> Yes it does. It defines as an alias, identical to what php does.

Great if it does, then your correction about `bool` is even less useful.
But actually I can't find it in [the proposal](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#abnf). Searching the entire document I find 0 instances of "boolean".